Submitted By: Ryan Oliver <ryan.oliver@pha.com.au>
Date: 2005-08-06
Initial Package Version: 3.1.11
Upstream Status: From Upstream
Origin: Upstream
Description: Patch release update

diff --recursive --unified libmikmod-3.1.11/NEWS libmikmod-3.1.11-a/NEWS
--- libmikmod-3.1.11/NEWS	2004-01-21 12:43:52.000000000 -0500
+++ libmikmod-3.1.11-a/NEWS	2004-06-04 08:24:26.000000000 -0400
@@ -1,3 +1,17 @@
+Summary of changes between libmikmod 3.1.11 and libmikmod 3.1.11-a
+========================================================================
+libmikmod-3.1.11-a was released on 05/03/2004
+
+Most of my work and attention goes on the 3.2.x series, but I still receive
+occasional bug reports and patchs for the 3.1.11 version. There are not enough
+changes to make a full release, but many people may find this patch useful.
+
+BUGFIXES
+- libmikmod playback now works correcly on amd64 (type size problem)
+- fixed warning issued by automake >= 1.8
+- config.sub and config.guess updated
+
+
 Summary of changes between libmikmod 3.1.10 and libmikmod 3.1.11 (Thiers):
 ==========================================================================
 
diff --recursive --unified libmikmod-3.1.11/config.guess libmikmod-3.1.11-a/config.guess
--- libmikmod-3.1.11/config.guess	2004-01-21 12:43:53.000000000 -0500
+++ libmikmod-3.1.11-a/config.guess	2004-06-04 08:24:26.000000000 -0400
@@ -1,9 +1,9 @@
 #! /bin/sh
 # Attempt to guess a canonical system name.
 #   Copyright (C) 1992, 1993, 1994, 1995, 1996, 1997, 1998, 1999,
-#   2000, 2001, 2002 Free Software Foundation, Inc.
+#   2000, 2001, 2002, 2003 Free Software Foundation, Inc.
 
-timestamp='2002-01-02'
+timestamp='2004-03-03'
 
 # This file is free software; you can redistribute it and/or modify it
 # under the terms of the GNU General Public License as published by
@@ -24,7 +24,7 @@
 # configuration script generated by Autoconf, you may include it under
 # the same distribution terms that you use for the rest of that program.
 
-# Originally written by Per Bothner <bothner@cygnus.com>.
+# Originally written by Per Bothner <per@bothner.com>.
 # Please send patches to <config-patches@gnu.org>.  Submit a context
 # diff and a properly formatted ChangeLog entry.
 #
@@ -88,30 +88,42 @@
   exit 1
 fi
 
+trap 'exit 1' 1 2 15
 
-dummy=dummy-$$
-trap 'rm -f $dummy.c $dummy.o $dummy.rel $dummy; exit 1' 1 2 15
+# CC_FOR_BUILD -- compiler used by this script. Note that the use of a
+# compiler to aid in system detection is discouraged as it requires
+# temporary files to be created and, as you can see below, it is a
+# headache to deal with in a portable fashion.
 
-# CC_FOR_BUILD -- compiler used by this script.
 # Historically, `CC_FOR_BUILD' used to be named `HOST_CC'. We still
 # use `HOST_CC' if defined, but it is deprecated.
 
-set_cc_for_build='case $CC_FOR_BUILD,$HOST_CC,$CC in
- ,,)    echo "int dummy(){}" > $dummy.c ;
-	for c in cc gcc c89 ; do
-	  ($c $dummy.c -c -o $dummy.o) >/dev/null 2>&1 ;
-	  if test $? = 0 ; then
+# Portable tmp directory creation inspired by the Autoconf team.
+
+set_cc_for_build='
+trap "exitcode=\$?; (rm -f \$tmpfiles 2>/dev/null; rmdir \$tmp 2>/dev/null) && exit \$exitcode" 0 ;
+trap "rm -f \$tmpfiles 2>/dev/null; rmdir \$tmp 2>/dev/null; exit 1" 1 2 13 15 ;
+: ${TMPDIR=/tmp} ;
+ { tmp=`(umask 077 && mktemp -d -q "$TMPDIR/cgXXXXXX") 2>/dev/null` && test -n "$tmp" && test -d "$tmp" ; } ||
+ { test -n "$RANDOM" && tmp=$TMPDIR/cg$$-$RANDOM && (umask 077 && mkdir $tmp) ; } ||
+ { tmp=$TMPDIR/cg-$$ && (umask 077 && mkdir $tmp) && echo "Warning: creating insecure temp directory" >&2 ; } ||
+ { echo "$me: cannot create a temporary directory in $TMPDIR" >&2 ; exit 1 ; } ;
+dummy=$tmp/dummy ;
+tmpfiles="$dummy.c $dummy.o $dummy.rel $dummy" ;
+case $CC_FOR_BUILD,$HOST_CC,$CC in
+ ,,)    echo "int x;" > $dummy.c ;
+	for c in cc gcc c89 c99 ; do
+	  if ($c -c -o $dummy.o $dummy.c) >/dev/null 2>&1 ; then
 	     CC_FOR_BUILD="$c"; break ;
 	  fi ;
 	done ;
-	rm -f $dummy.c $dummy.o $dummy.rel ;
 	if test x"$CC_FOR_BUILD" = x ; then
 	  CC_FOR_BUILD=no_compiler_found ;
 	fi
 	;;
  ,,*)   CC_FOR_BUILD=$CC ;;
  ,*,*)  CC_FOR_BUILD=$HOST_CC ;;
-esac'
+esac ;'
 
 # This is needed to find uname on a Pyramid OSx when run in the BSD universe.
 # (ghazi@noc.rutgers.edu 1994-08-24)
@@ -138,9 +150,11 @@
 	#
 	# Note: NetBSD doesn't particularly care about the vendor
 	# portion of the name.  We always set it to "unknown".
-	UNAME_MACHINE_ARCH=`(uname -p) 2>/dev/null` || \
-	    UNAME_MACHINE_ARCH=unknown
+	sysctl="sysctl -n hw.machine_arch"
+	UNAME_MACHINE_ARCH=`(/sbin/$sysctl 2>/dev/null || \
+	    /usr/sbin/$sysctl 2>/dev/null || echo unknown)`
 	case "${UNAME_MACHINE_ARCH}" in
+	    armeb) machine=armeb-unknown ;;
 	    arm*) machine=arm-unknown ;;
 	    sh3el) machine=shl-unknown ;;
 	    sh3eb) machine=sh-unknown ;;
@@ -166,18 +180,35 @@
 		;;
 	esac
 	# The OS release
-	release=`echo ${UNAME_RELEASE}|sed -e 's/[-_].*/\./'`
+	# Debian GNU/NetBSD machines have a different userland, and
+	# thus, need a distinct triplet. However, they do not need
+	# kernel version information, so it can be replaced with a
+	# suitable tag, in the style of linux-gnu.
+	case "${UNAME_VERSION}" in
+	    Debian*)
+		release='-gnu'
+		;;
+	    *)
+		release=`echo ${UNAME_RELEASE}|sed -e 's/[-_].*/\./'`
+		;;
+	esac
 	# Since CPU_TYPE-MANUFACTURER-KERNEL-OPERATING_SYSTEM:
 	# contains redundant information, the shorter form:
 	# CPU_TYPE-MANUFACTURER-OPERATING_SYSTEM is used.
 	echo "${machine}-${os}${release}"
 	exit 0 ;;
+    amd64:OpenBSD:*:*)
+	echo x86_64-unknown-openbsd${UNAME_RELEASE}
+	exit 0 ;;
     amiga:OpenBSD:*:*)
 	echo m68k-unknown-openbsd${UNAME_RELEASE}
 	exit 0 ;;
     arc:OpenBSD:*:*)
 	echo mipsel-unknown-openbsd${UNAME_RELEASE}
 	exit 0 ;;
+    cats:OpenBSD:*:*)
+	echo arm-unknown-openbsd${UNAME_RELEASE}
+	exit 0 ;;
     hp300:OpenBSD:*:*)
 	echo m68k-unknown-openbsd${UNAME_RELEASE}
 	exit 0 ;;
@@ -196,6 +227,9 @@
     mvmeppc:OpenBSD:*:*)
 	echo powerpc-unknown-openbsd${UNAME_RELEASE}
 	exit 0 ;;
+    pegasos:OpenBSD:*:*)
+	echo powerpc-unknown-openbsd${UNAME_RELEASE}
+	exit 0 ;;
     pmax:OpenBSD:*:*)
 	echo mipsel-unknown-openbsd${UNAME_RELEASE}
 	exit 0 ;;
@@ -211,68 +245,70 @@
     *:OpenBSD:*:*)
 	echo ${UNAME_MACHINE}-unknown-openbsd${UNAME_RELEASE}
 	exit 0 ;;
+    *:ekkoBSD:*:*)
+	echo ${UNAME_MACHINE}-unknown-ekkobsd${UNAME_RELEASE}
+	exit 0 ;;
+    macppc:MirBSD:*:*)
+	echo powerppc-unknown-mirbsd${UNAME_RELEASE}
+	exit 0 ;;
+    *:MirBSD:*:*)
+	echo ${UNAME_MACHINE}-unknown-mirbsd${UNAME_RELEASE}
+	exit 0 ;;
     alpha:OSF1:*:*)
-	if test $UNAME_RELEASE = "V4.0"; then
+	case $UNAME_RELEASE in
+	*4.0)
 		UNAME_RELEASE=`/usr/sbin/sizer -v | awk '{print $3}'`
-	fi
+		;;
+	*5.*)
+	        UNAME_RELEASE=`/usr/sbin/sizer -v | awk '{print $4}'`
+		;;
+	esac
+	# According to Compaq, /usr/sbin/psrinfo has been available on
+	# OSF/1 and Tru64 systems produced since 1995.  I hope that
+	# covers most systems running today.  This code pipes the CPU
+	# types through head -n 1, so we only detect the type of CPU 0.
+	ALPHA_CPU_TYPE=`/usr/sbin/psrinfo -v | sed -n -e 's/^  The alpha \(.*\) processor.*$/\1/p' | head -n 1`
+	case "$ALPHA_CPU_TYPE" in
+	    "EV4 (21064)")
+		UNAME_MACHINE="alpha" ;;
+	    "EV4.5 (21064)")
+		UNAME_MACHINE="alpha" ;;
+	    "LCA4 (21066/21068)")
+		UNAME_MACHINE="alpha" ;;
+	    "EV5 (21164)")
+		UNAME_MACHINE="alphaev5" ;;
+	    "EV5.6 (21164A)")
+		UNAME_MACHINE="alphaev56" ;;
+	    "EV5.6 (21164PC)")
+		UNAME_MACHINE="alphapca56" ;;
+	    "EV5.7 (21164PC)")
+		UNAME_MACHINE="alphapca57" ;;
+	    "EV6 (21264)")
+		UNAME_MACHINE="alphaev6" ;;
+	    "EV6.7 (21264A)")
+		UNAME_MACHINE="alphaev67" ;;
+	    "EV6.8CB (21264C)")
+		UNAME_MACHINE="alphaev68" ;;
+	    "EV6.8AL (21264B)")
+		UNAME_MACHINE="alphaev68" ;;
+	    "EV6.8CX (21264D)")
+		UNAME_MACHINE="alphaev68" ;;
+	    "EV6.9A (21264/EV69A)")
+		UNAME_MACHINE="alphaev69" ;;
+	    "EV7 (21364)")
+		UNAME_MACHINE="alphaev7" ;;
+	    "EV7.9 (21364A)")
+		UNAME_MACHINE="alphaev79" ;;
+	esac
+	# A Pn.n version is a patched version.
 	# A Vn.n version is a released version.
 	# A Tn.n version is a released field test version.
 	# A Xn.n version is an unreleased experimental baselevel.
 	# 1.2 uses "1.2" for uname -r.
-	cat <<EOF >$dummy.s
-	.data
-\$Lformat:
-	.byte 37,100,45,37,120,10,0	# "%d-%x\n"
-
-	.text
-	.globl main
-	.align 4
-	.ent main
-main:
-	.frame \$30,16,\$26,0
-	ldgp \$29,0(\$27)
-	.prologue 1
-	.long 0x47e03d80 # implver \$0
-	lda \$2,-1
-	.long 0x47e20c21 # amask \$2,\$1
-	lda \$16,\$Lformat
-	mov \$0,\$17
-	not \$1,\$18
-	jsr \$26,printf
-	ldgp \$29,0(\$26)
-	mov 0,\$16
-	jsr \$26,exit
-	.end main
-EOF
-	eval $set_cc_for_build
-	$CC_FOR_BUILD $dummy.s -o $dummy 2>/dev/null
-	if test "$?" = 0 ; then
-		case `./$dummy` in
-			0-0)
-				UNAME_MACHINE="alpha"
-				;;
-			1-0)
-				UNAME_MACHINE="alphaev5"
-				;;
-			1-1)
-				UNAME_MACHINE="alphaev56"
-				;;
-			1-101)
-				UNAME_MACHINE="alphapca56"
-				;;
-			2-303)
-				UNAME_MACHINE="alphaev6"
-				;;
-			2-307)
-				UNAME_MACHINE="alphaev67"
-				;;
-			2-1307)
-				UNAME_MACHINE="alphaev68"
-				;;
-		esac
-	fi
-	rm -f $dummy.s $dummy
-	echo ${UNAME_MACHINE}-dec-osf`echo ${UNAME_RELEASE} | sed -e 's/^[VTX]//' | tr 'ABCDEFGHIJKLMNOPQRSTUVWXYZ' 'abcdefghijklmnopqrstuvwxyz'`
+	echo ${UNAME_MACHINE}-dec-osf`echo ${UNAME_RELEASE} | sed -e 's/^[PVTX]//' | tr 'ABCDEFGHIJKLMNOPQRSTUVWXYZ' 'abcdefghijklmnopqrstuvwxyz'`
+	exit 0 ;;
+    Alpha*:OpenVMS:*:*)
+	echo alpha-hp-vms
 	exit 0 ;;
     Alpha\ *:Windows_NT*:*)
 	# How do we know it's Interix rather than the generic POSIX subsystem?
@@ -295,6 +331,9 @@
     *:OS/390:*:*)
 	echo i370-ibm-openedition
 	exit 0 ;;
+    *:OS400:*:*)
+        echo powerpc-ibm-os400
+	exit 0 ;;
     arm:RISC*:1.[012]*:*|arm:riscix:1.[012]*:*)
 	echo arm-acorn-riscix${UNAME_RELEASE}
 	exit 0;;
@@ -312,6 +351,13 @@
     NILE*:*:*:dcosx)
 	echo pyramid-pyramid-svr4
 	exit 0 ;;
+    DRS?6000:unix:4.0:6*)
+	echo sparc-icl-nx6
+	exit 0 ;;
+    DRS?6000:UNIX_SV:4.2*:7*)
+	case `/usr/bin/uname -p` in
+	    sparc) echo sparc-icl-nx7 && exit 0 ;;
+	esac ;;
     sun4H:SunOS:5.*:*)
 	echo sparc-hal-solaris2`echo ${UNAME_RELEASE}|sed -e 's/[^.]*//'`
 	exit 0 ;;
@@ -340,7 +386,7 @@
 	echo m68k-sun-sunos${UNAME_RELEASE}
 	exit 0 ;;
     sun*:*:4.2BSD:*)
-	UNAME_RELEASE=`(head -1 /etc/motd | awk '{print substr($5,1,3)}') 2>/dev/null`
+	UNAME_RELEASE=`(sed 1q /etc/motd | awk '{print substr($5,1,3)}') 2>/dev/null`
 	test "x${UNAME_RELEASE}" = "x" && UNAME_RELEASE=3
 	case "`/bin/arch`" in
 	    sun3)
@@ -380,6 +426,9 @@
     *:*MiNT:*:* | *:*mint:*:* | *:*TOS:*:*)
         echo m68k-unknown-mint${UNAME_RELEASE}
         exit 0 ;;
+    m68k:machten:*:*)
+	echo m68k-apple-machten${UNAME_RELEASE}
+	exit 0 ;;
     powerpc:machten:*:*)
 	echo powerpc-apple-machten${UNAME_RELEASE}
 	exit 0 ;;
@@ -418,15 +467,20 @@
 	  exit (-1);
 	}
 EOF
-	$CC_FOR_BUILD $dummy.c -o $dummy \
-	  && ./$dummy `echo "${UNAME_RELEASE}" | sed -n 's/\([0-9]*\).*/\1/p'` \
-	  && rm -f $dummy.c $dummy && exit 0
-	rm -f $dummy.c $dummy
+	$CC_FOR_BUILD -o $dummy $dummy.c \
+	  && $dummy `echo "${UNAME_RELEASE}" | sed -n 's/\([0-9]*\).*/\1/p'` \
+	  && exit 0
 	echo mips-mips-riscos${UNAME_RELEASE}
 	exit 0 ;;
     Motorola:PowerMAX_OS:*:*)
 	echo powerpc-motorola-powermax
 	exit 0 ;;
+    Motorola:*:4.3:PL8-*)
+	echo powerpc-harris-powermax
+	exit 0 ;;
+    Night_Hawk:*:*:PowerMAX_OS | Synergy:PowerMAX_OS:*:*)
+	echo powerpc-harris-powermax
+	exit 0 ;;
     Night_Hawk:Power_UNIX:*:*)
 	echo powerpc-harris-powerunix
 	exit 0 ;;
@@ -499,8 +553,7 @@
 			exit(0);
 			}
 EOF
-		$CC_FOR_BUILD $dummy.c -o $dummy && ./$dummy && rm -f $dummy.c $dummy && exit 0
-		rm -f $dummy.c $dummy
+		$CC_FOR_BUILD -o $dummy $dummy.c && $dummy && exit 0
 		echo rs6000-ibm-aix3.2.5
 	elif grep bos324 /usr/include/stdio.h >/dev/null 2>&1; then
 		echo rs6000-ibm-aix3.2.4
@@ -509,7 +562,7 @@
 	fi
 	exit 0 ;;
     *:AIX:*:[45])
-	IBM_CPU_ID=`/usr/sbin/lsdev -C -c processor -S available | head -1 | awk '{ print $1 }'`
+	IBM_CPU_ID=`/usr/sbin/lsdev -C -c processor -S available | sed 1q | awk '{ print $1 }'`
 	if /usr/sbin/lsattr -El ${IBM_CPU_ID} | grep ' POWER' >/dev/null 2>&1; then
 		IBM_ARCH=rs6000
 	else
@@ -598,11 +651,21 @@
                   exit (0);
               }
 EOF
-		    (CCOPTS= $CC_FOR_BUILD $dummy.c -o $dummy 2>/dev/null) && HP_ARCH=`./$dummy`
-		    if test -z "$HP_ARCH"; then HP_ARCH=hppa; fi
-		    rm -f $dummy.c $dummy
+		    (CCOPTS= $CC_FOR_BUILD -o $dummy $dummy.c 2>/dev/null) && HP_ARCH=`$dummy`
+		    test -z "$HP_ARCH" && HP_ARCH=hppa
 		fi ;;
 	esac
+	if [ ${HP_ARCH} = "hppa2.0w" ]
+	then
+	    # avoid double evaluation of $set_cc_for_build
+	    test -n "$CC_FOR_BUILD" || eval $set_cc_for_build
+	    if echo __LP64__ | (CCOPTS= $CC_FOR_BUILD -E -) | grep __LP64__ >/dev/null
+	    then
+		HP_ARCH="hppa2.0w"
+	    else
+		HP_ARCH="hppa64"
+	    fi
+	fi
 	echo ${HP_ARCH}-hp-hpux${HPUX_REV}
 	exit 0 ;;
     ia64:HP-UX:*:*)
@@ -636,8 +699,7 @@
 	  exit (0);
 	}
 EOF
-	$CC_FOR_BUILD $dummy.c -o $dummy && ./$dummy && rm -f $dummy.c $dummy && exit 0
-	rm -f $dummy.c $dummy
+	$CC_FOR_BUILD -o $dummy $dummy.c && $dummy && exit 0
 	echo unknown-hitachi-hiuxwe2
 	exit 0 ;;
     9000/7??:4.3bsd:*:* | 9000/8?[79]:4.3bsd:*:* )
@@ -683,9 +745,6 @@
     C4*:ConvexOS:*:* | convex:ConvexOS:C4*:*)
 	echo c4-convex-bsd
         exit 0 ;;
-    CRAY*X-MP:*:*:*)
-	echo xmp-cray-unicos
-        exit 0 ;;
     CRAY*Y-MP:*:*:*)
 	echo ymp-cray-unicos${UNAME_RELEASE} | sed -e 's/\.[^.]*$/.X/'
 	exit 0 ;;
@@ -698,24 +757,26 @@
     CRAY*TS:*:*:*)
 	echo t90-cray-unicos${UNAME_RELEASE} | sed -e 's/\.[^.]*$/.X/'
 	exit 0 ;;
-    CRAY*T3D:*:*:*)
-	echo alpha-cray-unicosmk${UNAME_RELEASE} | sed -e 's/\.[^.]*$/.X/'
-	exit 0 ;;
     CRAY*T3E:*:*:*)
 	echo alphaev5-cray-unicosmk${UNAME_RELEASE} | sed -e 's/\.[^.]*$/.X/'
 	exit 0 ;;
     CRAY*SV1:*:*:*)
 	echo sv1-cray-unicos${UNAME_RELEASE} | sed -e 's/\.[^.]*$/.X/'
 	exit 0 ;;
-    CRAY-2:*:*:*)
-	echo cray2-cray-unicos
-        exit 0 ;;
+    *:UNICOS/mp:*:*)
+	echo nv1-cray-unicosmp${UNAME_RELEASE} | sed -e 's/\.[^.]*$/.X/'
+	exit 0 ;;
     F30[01]:UNIX_System_V:*:* | F700:UNIX_System_V:*:*)
 	FUJITSU_PROC=`uname -m | tr 'ABCDEFGHIJKLMNOPQRSTUVWXYZ' 'abcdefghijklmnopqrstuvwxyz'`
         FUJITSU_SYS=`uname -p | tr 'ABCDEFGHIJKLMNOPQRSTUVWXYZ' 'abcdefghijklmnopqrstuvwxyz' | sed -e 's/\///'`
         FUJITSU_REL=`echo ${UNAME_RELEASE} | sed -e 's/ /_/'`
         echo "${FUJITSU_PROC}-fujitsu-${FUJITSU_SYS}${FUJITSU_REL}"
         exit 0 ;;
+    5000:UNIX_System_V:4.*:*)
+        FUJITSU_SYS=`uname -p | tr 'ABCDEFGHIJKLMNOPQRSTUVWXYZ' 'abcdefghijklmnopqrstuvwxyz' | sed -e 's/\///'`
+        FUJITSU_REL=`echo ${UNAME_RELEASE} | tr 'ABCDEFGHIJKLMNOPQRSTUVWXYZ' 'abcdefghijklmnopqrstuvwxyz' | sed -e 's/ /_/'`
+        echo "sparc-fujitsu-${FUJITSU_SYS}${FUJITSU_REL}"
+	exit 0 ;;
     i*86:BSD/386:*:* | i*86:BSD/OS:*:* | *:Ascend\ Embedded/OS:*:*)
 	echo ${UNAME_MACHINE}-pc-bsdi${UNAME_RELEASE}
 	exit 0 ;;
@@ -726,7 +787,21 @@
 	echo ${UNAME_MACHINE}-unknown-bsdi${UNAME_RELEASE}
 	exit 0 ;;
     *:FreeBSD:*:*)
-	echo ${UNAME_MACHINE}-unknown-freebsd`echo ${UNAME_RELEASE}|sed -e 's/[-(].*//'`
+	# Determine whether the default compiler uses glibc.
+	eval $set_cc_for_build
+	sed 's/^	//' << EOF >$dummy.c
+	#include <features.h>
+	#if __GLIBC__ >= 2
+	LIBC=gnu
+	#else
+	LIBC=
+	#endif
+EOF
+	eval `$CC_FOR_BUILD -E $dummy.c 2>/dev/null | grep ^LIBC=`
+	# GNU/KFreeBSD systems have a "k" prefix to indicate we are using
+	# FreeBSD's kernel, but not the complete OS.
+	case ${LIBC} in gnu) kernel_only='k' ;; esac
+	echo ${UNAME_MACHINE}-unknown-${kernel_only}freebsd`echo ${UNAME_RELEASE}|sed -e 's/[-(].*//'`${LIBC:+-$LIBC}
 	exit 0 ;;
     i*:CYGWIN*:*)
 	echo ${UNAME_MACHINE}-pc-cygwin
@@ -737,14 +812,17 @@
     i*:PW*:*)
 	echo ${UNAME_MACHINE}-pc-pw32
 	exit 0 ;;
-    x86:Interix*:3*)
-	echo i386-pc-interix3
+    x86:Interix*:[34]*)
+	echo i586-pc-interix${UNAME_RELEASE}|sed -e 's/\..*//'
+	exit 0 ;;
+    [345]86:Windows_95:* | [345]86:Windows_98:* | [345]86:Windows_NT:*)
+	echo i${UNAME_MACHINE}-pc-mks
 	exit 0 ;;
     i*:Windows_NT*:* | Pentium*:Windows_NT*:*)
 	# How do we know it's Interix rather than the generic POSIX subsystem?
 	# It also conflicts with pre-2.0 versions of AT&T UWIN. Should we
 	# UNAME_MACHINE based on the output of uname instead of i386?
-	echo i386-pc-interix
+	echo i586-pc-interix
 	exit 0 ;;
     i*:UWIN*:*)
 	echo ${UNAME_MACHINE}-pc-uwin
@@ -756,16 +834,24 @@
 	echo powerpcle-unknown-solaris2`echo ${UNAME_RELEASE}|sed -e 's/[^.]*//'`
 	exit 0 ;;
     *:GNU:*:*)
+	# the GNU system
 	echo `echo ${UNAME_MACHINE}|sed -e 's,[-/].*$,,'`-unknown-gnu`echo ${UNAME_RELEASE}|sed -e 's,/.*$,,'`
 	exit 0 ;;
+    *:GNU/*:*:*)
+	# other systems with GNU libc and userland
+	echo ${UNAME_MACHINE}-unknown-`echo ${UNAME_SYSTEM} | sed 's,^[^/]*/,,' | tr '[A-Z]' '[a-z]'``echo ${UNAME_RELEASE}|sed -e 's/[-(].*//'`-gnu
+	exit 0 ;;
     i*86:Minix:*:*)
 	echo ${UNAME_MACHINE}-pc-minix
 	exit 0 ;;
     arm*:Linux:*:*)
 	echo ${UNAME_MACHINE}-unknown-linux-gnu
 	exit 0 ;;
+    cris:Linux:*:*)
+	echo cris-axis-linux-gnu
+	exit 0 ;;
     ia64:Linux:*:*)
-	echo ${UNAME_MACHINE}-unknown-linux
+	echo ${UNAME_MACHINE}-unknown-linux-gnu
 	exit 0 ;;
     m68*:Linux:*:*)
 	echo ${UNAME_MACHINE}-unknown-linux-gnu
@@ -776,19 +862,37 @@
 	#undef CPU
 	#undef mips
 	#undef mipsel
-	#if defined(__MIPSEL__) || defined(__MIPSEL) || defined(_MIPSEL) || defined(MIPSEL) 
-	CPU=mipsel 
+	#if defined(__MIPSEL__) || defined(__MIPSEL) || defined(_MIPSEL) || defined(MIPSEL)
+	CPU=mipsel
 	#else
-	#if defined(__MIPSEB__) || defined(__MIPSEB) || defined(_MIPSEB) || defined(MIPSEB) 
+	#if defined(__MIPSEB__) || defined(__MIPSEB) || defined(_MIPSEB) || defined(MIPSEB)
 	CPU=mips
 	#else
 	CPU=
 	#endif
-	#endif 
+	#endif
+EOF
+	eval `$CC_FOR_BUILD -E $dummy.c 2>/dev/null | grep ^CPU=`
+	test x"${CPU}" != x && echo "${CPU}-unknown-linux-gnu" && exit 0
+	;;
+    mips64:Linux:*:*)
+	eval $set_cc_for_build
+	sed 's/^	//' << EOF >$dummy.c
+	#undef CPU
+	#undef mips64
+	#undef mips64el
+	#if defined(__MIPSEL__) || defined(__MIPSEL) || defined(_MIPSEL) || defined(MIPSEL)
+	CPU=mips64el
+	#else
+	#if defined(__MIPSEB__) || defined(__MIPSEB) || defined(_MIPSEB) || defined(MIPSEB)
+	CPU=mips64
+	#else
+	CPU=
+	#endif
+	#endif
 EOF
 	eval `$CC_FOR_BUILD -E $dummy.c 2>/dev/null | grep ^CPU=`
-	rm -f $dummy.c
-	test x"${CPU}" != x && echo "${CPU}-pc-linux-gnu" && exit 0
+	test x"${CPU}" != x && echo "${CPU}-unknown-linux-gnu" && exit 0
 	;;
     ppc:Linux:*:*)
 	echo powerpc-unknown-linux-gnu
@@ -824,6 +928,9 @@
     s390:Linux:*:* | s390x:Linux:*:*)
 	echo ${UNAME_MACHINE}-ibm-linux
 	exit 0 ;;
+    sh64*:Linux:*:*)
+    	echo ${UNAME_MACHINE}-unknown-linux-gnu
+	exit 0 ;;
     sh*:Linux:*:*)
 	echo ${UNAME_MACHINE}-unknown-linux-gnu
 	exit 0 ;;
@@ -837,7 +944,8 @@
 	# The BFD linker knows what the default object file format is, so
 	# first see if it will tell us. cd to the root directory to prevent
 	# problems with other programs or directories called `ld' in the path.
-	ld_supported_targets=`cd /; ld --help 2>&1 \
+	# Set LC_ALL=C to ensure ld outputs messages in English.
+	ld_supported_targets=`cd /; LC_ALL=C ld --help 2>&1 \
 			 | sed -ne '/supported targets:/!d
 				    s/[ 	][ 	]*/ /g
 				    s/.*supported targets: *//
@@ -849,7 +957,7 @@
 		;;
 	  a.out-i386-linux)
 		echo "${UNAME_MACHINE}-pc-linux-gnuaout"
-		exit 0 ;;		
+		exit 0 ;;
 	  coff-i386)
 		echo "${UNAME_MACHINE}-pc-linux-gnucoff"
 		exit 0 ;;
@@ -874,11 +982,17 @@
 	LIBC=gnulibc1
 	# endif
 	#else
+	#ifdef __INTEL_COMPILER
+	LIBC=gnu
+	#else
 	LIBC=gnuaout
 	#endif
+	#endif
+	#ifdef __dietlibc__
+	LIBC=dietlibc
+	#endif
 EOF
 	eval `$CC_FOR_BUILD -E $dummy.c 2>/dev/null | grep ^LIBC=`
-	rm -f $dummy.c
 	test x"${LIBC}" != x && echo "${UNAME_MACHINE}-pc-linux-${LIBC}" && exit 0
 	test x"${TENTATIVE}" != x && echo "${TENTATIVE}" && exit 0
 	;;
@@ -896,6 +1010,26 @@
         # Use sysv4.2uw... so that sysv4* matches it.
 	echo ${UNAME_MACHINE}-pc-sysv4.2uw${UNAME_VERSION}
 	exit 0 ;;
+    i*86:OS/2:*:*)
+	# If we were able to find `uname', then EMX Unix compatibility
+	# is probably installed.
+	echo ${UNAME_MACHINE}-pc-os2-emx
+	exit 0 ;;
+    i*86:XTS-300:*:STOP)
+	echo ${UNAME_MACHINE}-unknown-stop
+	exit 0 ;;
+    i*86:atheos:*:*)
+	echo ${UNAME_MACHINE}-unknown-atheos
+	exit 0 ;;
+	i*86:syllable:*:*)
+	echo ${UNAME_MACHINE}-pc-syllable
+	exit 0 ;;
+    i*86:LynxOS:2.*:* | i*86:LynxOS:3.[01]*:* | i*86:LynxOS:4.0*:*)
+	echo i386-unknown-lynxos${UNAME_RELEASE}
+	exit 0 ;;
+    i*86:*DOS:*:*)
+	echo ${UNAME_MACHINE}-pc-msdosdjgpp
+	exit 0 ;;
     i*86:*:4.*:* | i*86:SYSTEM_V:4.*:*)
 	UNAME_REL=`echo ${UNAME_RELEASE} | sed 's/\/MP$//'`
 	if grep Novell /usr/include/link.h >/dev/null 2>/dev/null; then
@@ -917,22 +1051,19 @@
 		UNAME_REL=`sed -n 's/.*Version //p' </usr/options/cb.name`
 		echo ${UNAME_MACHINE}-pc-isc$UNAME_REL
 	elif /bin/uname -X 2>/dev/null >/dev/null ; then
-		UNAME_REL=`(/bin/uname -X|egrep Release|sed -e 's/.*= //')`
-		(/bin/uname -X|egrep i80486 >/dev/null) && UNAME_MACHINE=i486
-		(/bin/uname -X|egrep '^Machine.*Pentium' >/dev/null) \
+		UNAME_REL=`(/bin/uname -X|grep Release|sed -e 's/.*= //')`
+		(/bin/uname -X|grep i80486 >/dev/null) && UNAME_MACHINE=i486
+		(/bin/uname -X|grep '^Machine.*Pentium' >/dev/null) \
 			&& UNAME_MACHINE=i586
-		(/bin/uname -X|egrep '^Machine.*Pent ?II' >/dev/null) \
+		(/bin/uname -X|grep '^Machine.*Pent *II' >/dev/null) \
 			&& UNAME_MACHINE=i686
-		(/bin/uname -X|egrep '^Machine.*Pentium Pro' >/dev/null) \
+		(/bin/uname -X|grep '^Machine.*Pentium Pro' >/dev/null) \
 			&& UNAME_MACHINE=i686
 		echo ${UNAME_MACHINE}-pc-sco$UNAME_REL
 	else
 		echo ${UNAME_MACHINE}-pc-sysv32
 	fi
 	exit 0 ;;
-    i*86:*DOS:*:*)
-	echo ${UNAME_MACHINE}-pc-msdosdjgpp
-	exit 0 ;;
     pc:*:*:*)
 	# Left here for compatibility:
         # uname -m prints for DJGPP always 'pc', but it prints nothing about
@@ -956,9 +1087,15 @@
 	# "miniframe"
 	echo m68010-convergent-sysv
 	exit 0 ;;
+    mc68k:UNIX:SYSTEM5:3.51m)
+	echo m68k-convergent-sysv
+	exit 0 ;;
+    M680?0:D-NIX:5.3:*)
+	echo m68k-diab-dnix
+	exit 0 ;;
     M68*:*:R3V[567]*:*)
 	test -r /sysV68 && echo 'm68k-motorola-sysv' && exit 0 ;;
-    3[34]??:*:4.0:3.0 | 3[34]??A:*:4.0:3.0 | 3[34]??,*:*:4.0:3.0 | 3[34]??/*:*:4.0:3.0 | 4850:*:4.0:3.0 | SKA40:*:4.0:3.0)
+    3[345]??:*:4.0:3.0 | 3[34]??A:*:4.0:3.0 | 3[34]??,*:*:4.0:3.0 | 3[34]??/*:*:4.0:3.0 | 4400:*:4.0:3.0 | 4850:*:4.0:3.0 | SKA40:*:4.0:3.0 | SDS2:*:4.0:3.0 | SHG2:*:4.0:3.0)
 	OS_REL=''
 	test -r /etc/.relid \
 	&& OS_REL=.`sed -n 's/[^ ]* [^ ]* \([0-9][0-9]\).*/\1/p' < /etc/.relid`
@@ -975,9 +1112,6 @@
     mc68030:UNIX_System_V:4.*:*)
 	echo m68k-atari-sysv4
 	exit 0 ;;
-    i*86:LynxOS:2.*:* | i*86:LynxOS:3.[01]*:* | i*86:LynxOS:4.0*:*)
-	echo i386-unknown-lynxos${UNAME_RELEASE}
-	exit 0 ;;
     TSUNAMI:LynxOS:2.*:*)
 	echo sparc-unknown-lynxos${UNAME_RELEASE}
 	exit 0 ;;
@@ -1049,6 +1183,9 @@
     SX-5:SUPER-UX:*:*)
 	echo sx5-nec-superux${UNAME_RELEASE}
 	exit 0 ;;
+    SX-6:SUPER-UX:*:*)
+	echo sx6-nec-superux${UNAME_RELEASE}
+	exit 0 ;;
     Power*:Rhapsody:*:*)
 	echo powerpc-apple-rhapsody${UNAME_RELEASE}
 	exit 0 ;;
@@ -1056,18 +1193,24 @@
 	echo ${UNAME_MACHINE}-apple-rhapsody${UNAME_RELEASE}
 	exit 0 ;;
     *:Darwin:*:*)
-	echo `uname -p`-apple-darwin${UNAME_RELEASE}
+	case `uname -p` in
+	    *86) UNAME_PROCESSOR=i686 ;;
+	    powerpc) UNAME_PROCESSOR=powerpc ;;
+	esac
+	echo ${UNAME_PROCESSOR}-apple-darwin${UNAME_RELEASE}
 	exit 0 ;;
     *:procnto*:*:* | *:QNX:[0123456789]*:*)
-	if test "${UNAME_MACHINE}" = "x86pc"; then
+	UNAME_PROCESSOR=`uname -p`
+	if test "$UNAME_PROCESSOR" = "x86"; then
+		UNAME_PROCESSOR=i386
 		UNAME_MACHINE=pc
 	fi
-	echo `uname -p`-${UNAME_MACHINE}-nto-qnx
+	echo ${UNAME_PROCESSOR}-${UNAME_MACHINE}-nto-qnx${UNAME_RELEASE}
 	exit 0 ;;
     *:QNX:*:4*)
 	echo i386-pc-qnx
 	exit 0 ;;
-    NSR-[GKLNPTVW]:NONSTOP_KERNEL:*:*)
+    NSR-?:NONSTOP_KERNEL:*:*)
 	echo nsr-tandem-nsk${UNAME_RELEASE}
 	exit 0 ;;
     *:NonStop-UX:*:*)
@@ -1090,11 +1233,6 @@
 	fi
 	echo ${UNAME_MACHINE}-unknown-plan9
 	exit 0 ;;
-    i*86:OS/2:*:*)
-	# If we were able to find `uname', then EMX Unix compatibility
-	# is probably installed.
-	echo ${UNAME_MACHINE}-pc-os2-emx
-	exit 0 ;;
     *:TOPS-10:*:*)
 	echo pdp10-unknown-tops10
 	exit 0 ;;
@@ -1113,11 +1251,11 @@
     *:ITS:*:*)
 	echo pdp10-unknown-its
 	exit 0 ;;
-    i*86:XTS-300:*:STOP)
-	echo ${UNAME_MACHINE}-unknown-stop
+    SEI:*:*:SEIUX)
+        echo mips-sei-seiux${UNAME_RELEASE}
 	exit 0 ;;
-    i*86:atheos:*:*)
-	echo ${UNAME_MACHINE}-unknown-atheos
+    *:DragonFly:*:*)
+	echo ${UNAME_MACHINE}-unknown-dragonfly`echo ${UNAME_RELEASE}|sed -e 's/[-(].*//'`
 	exit 0 ;;
 esac
 
@@ -1239,8 +1377,7 @@
 }
 EOF
 
-$CC_FOR_BUILD $dummy.c -o $dummy 2>/dev/null && ./$dummy && rm -f $dummy.c $dummy && exit 0
-rm -f $dummy.c $dummy
+$CC_FOR_BUILD -o $dummy $dummy.c 2>/dev/null && $dummy && exit 0
 
 # Apollos put the system type in the environment.
 
diff --recursive --unified libmikmod-3.1.11/config.sub libmikmod-3.1.11-a/config.sub
--- libmikmod-3.1.11/config.sub	2004-01-21 12:43:53.000000000 -0500
+++ libmikmod-3.1.11-a/config.sub	2004-06-04 08:24:26.000000000 -0400
@@ -3,7 +3,7 @@
 #   Copyright (C) 1992, 1993, 1994, 1995, 1996, 1997, 1998, 1999,
 #   2000, 2001, 2002 Free Software Foundation, Inc.
 
-timestamp='2002-01-02'
+timestamp='2002-07-03'
 
 # This file is (in principle) common to ALL GNU software.
 # The presence of a machine in this file suggests that SOME GNU software
@@ -118,7 +118,7 @@
 # Here we must recognize all the valid KERNEL-OS combinations.
 maybe_os=`echo $1 | sed 's/^\(.*\)-\([^-]*-[^-]*\)$/\2/'`
 case $maybe_os in
-  nto-qnx* | linux-gnu* | storm-chaos* | os2-emx* | windows32-*)
+  nto-qnx* | linux-gnu* | freebsd*-gnu* | storm-chaos* | os2-emx* | windows32-* | rtmk-nova*)
     os=-$maybe_os
     basic_machine=`echo $1 | sed 's/^\(.*\)-\([^-]*-[^-]*\)$/\1/'`
     ;;
@@ -230,24 +230,32 @@
 	| alpha64 | alpha64ev[4-8] | alpha64ev56 | alpha64ev6[78] | alpha64pca5[67] \
 	| arc | arm | arm[bl]e | arme[lb] | armv[2345] | armv[345][lb] | avr \
 	| c4x | clipper \
-	| d10v | d30v | dsp16xx \
-	| fr30 \
+	| d10v | d30v | dlx | dsp16xx \
+	| fr30 | frv \
 	| h8300 | h8500 | hppa | hppa1.[01] | hppa2.0 | hppa2.0[nw] | hppa64 \
 	| i370 | i860 | i960 | ia64 \
+	| ip2k \
 	| m32r | m68000 | m68k | m88k | mcore \
-	| mips16 | mips64 | mips64el | mips64orion | mips64orionel \
-	| mips64vr4100 | mips64vr4100el | mips64vr4300 \
-	| mips64vr4300el | mips64vr5000 | mips64vr5000el \
-	| mipsbe | mipseb | mipsel | mipsle | mipstx39 | mipstx39el \
-	| mipsisa32 \
+	| mips | mipsbe | mipseb | mipsel | mipsle \
+	| mips16 \
+	| mips64 | mips64el \
+	| mips64orion | mips64orionel \
+	| mips64vr4100 | mips64vr4100el \
+	| mips64vr4300 | mips64vr4300el \
+	| mips64vr5000 | mips64vr5000el \
+	| mipsisa32 | mipsisa32el \
+	| mipsisa64 | mipsisa64el \
+	| mipsisa64sb1 | mipsisa64sb1el \
+	| mipstx39 | mipstx39el \
 	| mn10200 | mn10300 \
 	| ns16k | ns32k \
-	| openrisc \
+	| openrisc | or32 \
 	| pdp10 | pdp11 | pj | pjl \
 	| powerpc | powerpc64 | powerpc64le | powerpcle | ppcbe \
 	| pyramid \
-	| sh | sh[34] | sh[34]eb | shbe | shle \
-	| sparc | sparc64 | sparclet | sparclite | sparcv9 | sparcv9b \
+	| sh | sh[1234] | sh3e | sh[34]eb | shbe | shle | sh[1234]le | sh3ele \
+	| sh64 | sh64le \
+	| sparc | sparc64 | sparc86x | sparclet | sparclite | sparcv9 | sparcv9b \
 	| strongarm \
 	| tahoe | thumb | tic80 | tron \
 	| v850 | v850e \
@@ -281,37 +289,46 @@
 	| alpha-* | alphaev[4-8]-* | alphaev56-* | alphaev6[78]-* \
 	| alpha64-* | alpha64ev[4-8]-* | alpha64ev56-* | alpha64ev6[78]-* \
 	| alphapca5[67]-* | alpha64pca5[67]-* | arc-* \
-	| arm-*  | armbe-* | armle-* | armv*-* \
+	| arm-*  | armbe-* | armle-* | armeb-* | armv*-* \
 	| avr-* \
 	| bs2000-* \
 	| c[123]* | c30-* | [cjt]90-* | c54x-* \
-	| clipper-* | cray2-* | cydra-* \
-	| d10v-* | d30v-* \
+	| clipper-* | cydra-* \
+	| d10v-* | d30v-* | dlx-* \
 	| elxsi-* \
-	| f30[01]-* | f700-* | fr30-* | fx80-* \
+	| f30[01]-* | f700-* | fr30-* | frv-* | fx80-* \
 	| h8300-* | h8500-* \
 	| hppa-* | hppa1.[01]-* | hppa2.0-* | hppa2.0[nw]-* | hppa64-* \
 	| i*86-* | i860-* | i960-* | ia64-* \
+	| ip2k-* \
 	| m32r-* \
-	| m68000-* | m680[01234]0-* | m68360-* | m683?2-* | m68k-* \
+	| m68000-* | m680[012346]0-* | m68360-* | m683?2-* | m68k-* \
 	| m88110-* | m88k-* | mcore-* \
-	| mips-* | mips16-* | mips64-* | mips64el-* | mips64orion-* \
-	| mips64orionel-* | mips64vr4100-* | mips64vr4100el-* \
-	| mips64vr4300-* | mips64vr4300el-* | mipsbe-* | mipseb-* \
-	| mipsle-* | mipsel-* | mipstx39-* | mipstx39el-* \
+	| mips-* | mipsbe-* | mipseb-* | mipsel-* | mipsle-* \
+	| mips16-* \
+	| mips64-* | mips64el-* \
+	| mips64orion-* | mips64orionel-* \
+	| mips64vr4100-* | mips64vr4100el-* \
+	| mips64vr4300-* | mips64vr4300el-* \
+	| mips64vr5000-* | mips64vr5000el-* \
+	| mipsisa32-* | mipsisa32el-* \
+	| mipsisa64-* | mipsisa64el-* \
+	| mipsisa64sb1-* | mipsisa64sb1el-* \
+	| mipstx39 | mipstx39el \
 	| none-* | np1-* | ns16k-* | ns32k-* \
 	| orion-* \
 	| pdp10-* | pdp11-* | pj-* | pjl-* | pn-* | power-* \
 	| powerpc-* | powerpc64-* | powerpc64le-* | powerpcle-* | ppcbe-* \
 	| pyramid-* \
 	| romp-* | rs6000-* \
-	| sh-* | sh[34]-* | sh[34]eb-* | shbe-* | shle-* \
-	| sparc-* | sparc64-* | sparc86x-* | sparclite-* \
-	| sparcv9-* | sparcv9b-* | strongarm-* | sv1-* \
-	| t3e-* | tahoe-* | thumb-* | tic30-* | tic54x-* | tic80-* | tron-* \
+	| sh-* | sh[1234]-* | sh3e-* | sh[34]eb-* | shbe-* \
+	| shle-* | sh[1234]le-* | sh3ele-* | sh64-* | sh64le-* \
+	| sparc-* | sparc64-* | sparc86x-* | sparclet-* | sparclite-* \
+	| sparcv9-* | sparcv9b-* | strongarm-* | sv1-* | sx?-* \
+	| tahoe-* | thumb-* | tic30-* | tic54x-* | tic80-* | tron-* \
 	| v850-* | v850e-* | vax-* \
 	| we32k-* \
-	| x86-* | x86_64-* | xmp-* | xps100-* | xscale-* | xstormy16-* \
+	| x86-* | x86_64-* | xps100-* | xscale-* | xstormy16-* \
 	| xtensa-* \
 	| ymp-* \
 	| z8k-*)
@@ -377,6 +394,10 @@
 		basic_machine=ns32k-sequent
 		os=-dynix
 		;;
+	c90)
+		basic_machine=c90-cray
+		os=-unicos
+		;;
 	convex-c1)
 		basic_machine=c1-convex
 		os=-bsd
@@ -397,16 +418,8 @@
 		basic_machine=c38-convex
 		os=-bsd
 		;;
-	cray | ymp)
-		basic_machine=ymp-cray
-		os=-unicos
-		;;
-	cray2)
-		basic_machine=cray2-cray
-		os=-unicos
-		;;
-	[cjt]90)
-		basic_machine=${basic_machine}-cray
+	cray | j90)
+		basic_machine=j90-cray
 		os=-unicos
 		;;
 	crds | unos)
@@ -609,14 +622,6 @@
 		basic_machine=m68k-atari
 		os=-mint
 		;;
-	mipsel*-linux*)
-		basic_machine=mipsel-unknown
-		os=-linux-gnu
-		;;
-	mips*-linux*)
-		basic_machine=mips-unknown
-		os=-linux-gnu
-		;;
 	mips3*-*)
 		basic_machine=`echo $basic_machine | sed -e 's/mips3/mips64/'`
 		;;
@@ -714,6 +719,10 @@
 		basic_machine=hppa1.1-oki
 		os=-proelf
 		;;
+	or32 | or32-*)
+		basic_machine=or32-unknown
+		os=-coff
+		;;
 	OSE68000 | ose68000)
 		basic_machine=m68000-ericsson
 		os=-ose
@@ -874,7 +883,7 @@
 	sun386 | sun386i | roadrunner)
 		basic_machine=i386-sun
 		;;
-	sv1)
+        sv1)
 		basic_machine=sv1-cray
 		os=-unicos
 		;;
@@ -882,8 +891,16 @@
 		basic_machine=i386-sequent
 		os=-dynix
 		;;
+	t3d)
+		basic_machine=alpha-cray
+		os=-unicos
+		;;
 	t3e)
-		basic_machine=t3e-cray
+		basic_machine=alphaev5-cray
+		os=-unicos
+		;;
+	t90)
+		basic_machine=t90-cray
 		os=-unicos
 		;;
 	tic54x | c54x*)
@@ -950,13 +967,13 @@
 		basic_machine=i386-pc
 		os=-windows32-msvcrt
 		;;
-	xmp)
-		basic_machine=xmp-cray
-		os=-unicos
-		;;
         xps | xps100)
 		basic_machine=xps100-honeywell
 		;;
+	ymp)
+		basic_machine=ymp-cray
+		os=-unicos
+		;;
 	z8k-*-coff)
 		basic_machine=z8k-unknown
 		os=-sim
@@ -977,13 +994,6 @@
 	op60c)
 		basic_machine=hppa1.1-oki
 		;;
-	mips)
-		if [ x$os = x-linux-gnu ]; then
-			basic_machine=mips-unknown
-		else
-			basic_machine=mips-mips
-		fi
-		;;
 	romp)
 		basic_machine=romp-ibm
 		;;
@@ -1003,9 +1013,12 @@
 	we32k)
 		basic_machine=we32k-att
 		;;
-	sh3 | sh4 | sh3eb | sh4eb)
+	sh3 | sh4 | sh3eb | sh4eb | sh[1234]le | sh3ele)
 		basic_machine=sh-unknown
 		;;
+	sh64)
+		basic_machine=sh64-unknown
+		;;
 	sparc | sparcv9 | sparcv9b)
 		basic_machine=sparc-sun
 		;;
@@ -1093,7 +1106,8 @@
 	      | -interix* | -uwin* | -rhapsody* | -darwin* | -opened* \
 	      | -openstep* | -oskit* | -conix* | -pw32* | -nonstopux* \
 	      | -storm-chaos* | -tops10* | -tenex* | -tops20* | -its* \
-	      | -os2* | -vos* | -palmos* | -uclinux* | -nucleus* | -morphos*)
+	      | -os2* | -vos* | -palmos* | -uclinux* | -nucleus* \
+	      | -morphos* | -superux* | -rtmk* | -rtmk-nova* | -windiss* | -powermax*)
 	# Remember, each alternative MUST END IN *, to match a version number.
 		;;
 	-qnx*)
@@ -1154,6 +1168,9 @@
 	-ctix* | -uts*)
 		os=-sysv
 		;;
+	-nova*)
+		os=-rtmk-nova
+		;;
 	-ns2 )
 	        os=-nextstep2
 		;;
@@ -1259,6 +1276,9 @@
 	mips*-*)
 		os=-elf
 		;;
+	or32-*)
+		os=-coff
+		;;
 	*-tti)	# must be before sparc entry or we get the wrong os.
 		os=-sysv3
 		;;
@@ -1406,7 +1426,7 @@
 			-ptx*)
 				vendor=sequent
 				;;
-			-vxsim* | -vxworks*)
+			-vxsim* | -vxworks* | -windiss*)
 				vendor=wrs
 				;;
 			-aux*)
diff --recursive --unified libmikmod-3.1.11/configure.in libmikmod-3.1.11-a/configure.in
--- libmikmod-3.1.11/configure.in	2004-01-21 12:43:52.000000000 -0500
+++ libmikmod-3.1.11-a/configure.in	2004-06-04 08:24:26.000000000 -0400
@@ -1,6 +1,6 @@
 dnl Process this file with autoconf to produce a configure script.
 dnl
-dnl $Id: configure.in,v 1.2 2004/01/21 01:41:49 raph Exp $
+dnl $Id: configure.in,v 1.1.1.1 2004/06/01 12:16:16 raph Exp $
 
 AC_INIT(include/mikmod.h.in)
 
diff --recursive --unified libmikmod-3.1.11/dlapi/dl_hpux.c libmikmod-3.1.11-a/dlapi/dl_hpux.c
--- libmikmod-3.1.11/dlapi/dl_hpux.c	2004-01-21 12:43:52.000000000 -0500
+++ libmikmod-3.1.11-a/dlapi/dl_hpux.c	2004-06-04 08:24:26.000000000 -0400
@@ -1,6 +1,6 @@
 /*==============================================================================
 
-  $Id: dl_hpux.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: dl_hpux.c,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
 
   simple dlopen()-like implementation above HP-UX shl_xxx() API
 
diff --recursive --unified libmikmod-3.1.11/docs/mikmod.texi libmikmod-3.1.11-a/docs/mikmod.texi
--- libmikmod-3.1.11/docs/mikmod.texi	2004-01-21 12:43:52.000000000 -0500
+++ libmikmod-3.1.11-a/docs/mikmod.texi	2004-06-04 08:24:26.000000000 -0400
@@ -7,7 +7,7 @@
 
   MikMod Sound Library Documentation
 
-  $Id: mikmod.texi,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: mikmod.texi,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
 
 @end ignore
 @c comment this during modifications
diff --recursive --unified libmikmod-3.1.11/drivers/drv_AF.c libmikmod-3.1.11-a/drivers/drv_AF.c
--- libmikmod-3.1.11/drivers/drv_AF.c	2004-01-21 12:43:52.000000000 -0500
+++ libmikmod-3.1.11-a/drivers/drv_AF.c	2004-06-04 08:24:26.000000000 -0400
@@ -20,7 +20,7 @@
   
 /*==============================================================================
 
-  $Id: drv_AF.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: drv_AF.c,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
 
   Driver for output on AF audio server.
 
diff --recursive --unified libmikmod-3.1.11/drivers/drv_aix.c libmikmod-3.1.11-a/drivers/drv_aix.c
--- libmikmod-3.1.11/drivers/drv_aix.c	2004-01-21 12:43:52.000000000 -0500
+++ libmikmod-3.1.11-a/drivers/drv_aix.c	2004-06-04 08:24:26.000000000 -0400
@@ -20,7 +20,7 @@
   
 /*==============================================================================
 
-  $Id: drv_aix.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: drv_aix.c,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
 
   Driver for output to AIX series audio device
 
diff --recursive --unified libmikmod-3.1.11/drivers/drv_alsa.c libmikmod-3.1.11-a/drivers/drv_alsa.c
--- libmikmod-3.1.11/drivers/drv_alsa.c	2004-01-21 12:43:52.000000000 -0500
+++ libmikmod-3.1.11-a/drivers/drv_alsa.c	2004-06-04 08:24:26.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: drv_alsa.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: drv_alsa.c,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
 
   Driver for Advanced Linux Sound Architecture (ALSA)
 
diff --recursive --unified libmikmod-3.1.11/drivers/drv_dart.c libmikmod-3.1.11-a/drivers/drv_dart.c
--- libmikmod-3.1.11/drivers/drv_dart.c	2004-01-21 12:43:52.000000000 -0500
+++ libmikmod-3.1.11-a/drivers/drv_dart.c	2004-06-04 08:24:26.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: drv_dart.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: drv_dart.c,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
 
   Driver for output on OS/2 MMPM/2 using direct audio (DART)
 
diff --recursive --unified libmikmod-3.1.11/drivers/drv_ds.c libmikmod-3.1.11-a/drivers/drv_ds.c
--- libmikmod-3.1.11/drivers/drv_ds.c	2004-01-21 12:43:52.000000000 -0500
+++ libmikmod-3.1.11-a/drivers/drv_ds.c	2004-06-04 08:24:26.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: drv_ds.c,v 1.2 2004/01/21 17:29:32 raph Exp $
+  $Id: drv_ds.c,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
 
   Driver for output on win32 platforms using DirectSound
 
diff --recursive --unified libmikmod-3.1.11/drivers/drv_esd.c libmikmod-3.1.11-a/drivers/drv_esd.c
--- libmikmod-3.1.11/drivers/drv_esd.c	2004-01-21 12:43:52.000000000 -0500
+++ libmikmod-3.1.11-a/drivers/drv_esd.c	2004-06-04 08:24:26.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: drv_esd.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: drv_esd.c,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
 
   Driver for the Enlightened sound daemon (EsounD)
 
diff --recursive --unified libmikmod-3.1.11/drivers/drv_hp.c libmikmod-3.1.11-a/drivers/drv_hp.c
--- libmikmod-3.1.11/drivers/drv_hp.c	2004-01-21 12:43:52.000000000 -0500
+++ libmikmod-3.1.11-a/drivers/drv_hp.c	2004-06-04 08:24:26.000000000 -0400
@@ -20,7 +20,7 @@
   
 /*==============================================================================
 
-  $Id: drv_hp.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: drv_hp.c,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
 
   Driver for output to HP 9000 series /dev/audio
 
diff --recursive --unified libmikmod-3.1.11/drivers/drv_mac.c libmikmod-3.1.11-a/drivers/drv_mac.c
--- libmikmod-3.1.11/drivers/drv_mac.c	2004-01-21 12:43:52.000000000 -0500
+++ libmikmod-3.1.11-a/drivers/drv_mac.c	2004-06-04 08:24:26.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: drv_mac.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: drv_mac.c,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
 
   Driver for output to the Macintosh Sound Manager
 
diff --recursive --unified libmikmod-3.1.11/drivers/drv_nos.c libmikmod-3.1.11-a/drivers/drv_nos.c
--- libmikmod-3.1.11/drivers/drv_nos.c	2004-01-21 12:43:52.000000000 -0500
+++ libmikmod-3.1.11-a/drivers/drv_nos.c	2004-06-04 08:24:26.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: drv_nos.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: drv_nos.c,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
 
   Driver for no output
 
diff --recursive --unified libmikmod-3.1.11/drivers/drv_os2.c libmikmod-3.1.11-a/drivers/drv_os2.c
--- libmikmod-3.1.11/drivers/drv_os2.c	2004-01-21 12:43:52.000000000 -0500
+++ libmikmod-3.1.11-a/drivers/drv_os2.c	2004-06-04 08:24:26.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: drv_os2.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: drv_os2.c,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
 
   Driver for output on OS/2 using MMPM/2 MCI interface
 
diff --recursive --unified libmikmod-3.1.11/drivers/drv_oss.c libmikmod-3.1.11-a/drivers/drv_oss.c
--- libmikmod-3.1.11/drivers/drv_oss.c	2004-01-21 12:43:52.000000000 -0500
+++ libmikmod-3.1.11-a/drivers/drv_oss.c	2004-06-04 08:24:26.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: drv_oss.c,v 1.2 2004/01/21 01:41:53 raph Exp $
+  $Id: drv_oss.c,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
 
   Driver for output on Linux and FreeBSD Open Sound System (OSS) (/dev/dsp) 
 
diff --recursive --unified libmikmod-3.1.11/drivers/drv_pipe.c libmikmod-3.1.11-a/drivers/drv_pipe.c
--- libmikmod-3.1.11/drivers/drv_pipe.c	2004-01-21 12:43:52.000000000 -0500
+++ libmikmod-3.1.11-a/drivers/drv_pipe.c	2004-06-04 08:24:26.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: drv_pipe.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: drv_pipe.c,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
 
   Driver for output via a pipe to another command
 
diff --recursive --unified libmikmod-3.1.11/drivers/drv_raw.c libmikmod-3.1.11-a/drivers/drv_raw.c
--- libmikmod-3.1.11/drivers/drv_raw.c	2004-01-21 12:43:52.000000000 -0500
+++ libmikmod-3.1.11-a/drivers/drv_raw.c	2004-06-04 08:24:26.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: drv_raw.c,v 1.2 2004/01/21 17:29:32 raph Exp $
+  $Id: drv_raw.c,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
 
   Driver for output to a file called MUSIC.RAW
 
diff --recursive --unified libmikmod-3.1.11/drivers/drv_sam9407.c libmikmod-3.1.11-a/drivers/drv_sam9407.c
--- libmikmod-3.1.11/drivers/drv_sam9407.c	2004-01-21 12:43:52.000000000 -0500
+++ libmikmod-3.1.11-a/drivers/drv_sam9407.c	2004-06-04 08:24:26.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
   
-  $Id: drv_sam9407.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: drv_sam9407.c,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
   
   Driver for the Linux sam9407 driver
   
diff --recursive --unified libmikmod-3.1.11/drivers/drv_sgi.c libmikmod-3.1.11-a/drivers/drv_sgi.c
--- libmikmod-3.1.11/drivers/drv_sgi.c	2004-01-21 12:43:52.000000000 -0500
+++ libmikmod-3.1.11-a/drivers/drv_sgi.c	2004-06-04 08:24:26.000000000 -0400
@@ -20,7 +20,7 @@
   
 /*==============================================================================
 
-  $Id: drv_sgi.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: drv_sgi.c,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
 
   Driver for output on SGI audio system (needs libaudio from the dmedia
   package).
diff --recursive --unified libmikmod-3.1.11/drivers/drv_stdout.c libmikmod-3.1.11-a/drivers/drv_stdout.c
--- libmikmod-3.1.11/drivers/drv_stdout.c	2004-01-21 12:43:52.000000000 -0500
+++ libmikmod-3.1.11-a/drivers/drv_stdout.c	2004-06-04 08:24:26.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: drv_stdout.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: drv_stdout.c,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
 
   Output data to stdout
 
diff --recursive --unified libmikmod-3.1.11/drivers/drv_sun.c libmikmod-3.1.11-a/drivers/drv_sun.c
--- libmikmod-3.1.11/drivers/drv_sun.c	2004-01-21 12:43:52.000000000 -0500
+++ libmikmod-3.1.11-a/drivers/drv_sun.c	2004-06-04 08:24:26.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: drv_sun.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: drv_sun.c,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
 
   Driver for output on the Sun audio device (/dev/audio).
   Also works under NetBSD and OpenBSD
diff --recursive --unified libmikmod-3.1.11/drivers/drv_ultra.c libmikmod-3.1.11-a/drivers/drv_ultra.c
--- libmikmod-3.1.11/drivers/drv_ultra.c	2004-01-21 12:43:52.000000000 -0500
+++ libmikmod-3.1.11-a/drivers/drv_ultra.c	2004-06-04 08:24:26.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: drv_ultra.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: drv_ultra.c,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
 
   Driver for the Linux Ultrasound driver
 
diff --recursive --unified libmikmod-3.1.11/drivers/drv_wav.c libmikmod-3.1.11-a/drivers/drv_wav.c
--- libmikmod-3.1.11/drivers/drv_wav.c	2004-01-21 12:43:52.000000000 -0500
+++ libmikmod-3.1.11-a/drivers/drv_wav.c	2004-06-04 08:24:26.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: drv_wav.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: drv_wav.c,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
 
   Driver for output to a file called MUSIC.WAV
 
diff --recursive --unified libmikmod-3.1.11/drivers/drv_win.c libmikmod-3.1.11-a/drivers/drv_win.c
--- libmikmod-3.1.11/drivers/drv_win.c	2004-01-21 12:43:52.000000000 -0500
+++ libmikmod-3.1.11-a/drivers/drv_win.c	2004-06-04 08:24:26.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: drv_win.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: drv_win.c,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
 
   Driver for output on win32 platforms using the multimedia API
 
diff --recursive --unified libmikmod-3.1.11/include/mikmod.h.in libmikmod-3.1.11-a/include/mikmod.h.in
--- libmikmod-3.1.11/include/mikmod.h.in	2004-01-21 12:43:53.000000000 -0500
+++ libmikmod-3.1.11-a/include/mikmod.h.in	2004-06-04 08:24:26.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: mikmod.h.in,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: mikmod.h.in,v 1.2 2004/06/01 16:43:45 raph Exp $
 
   MikMod sound library include file
 
@@ -85,7 +85,7 @@
 
 @DOES_NOT_HAVE_SIGNED@
 
-#if defined(__arch64__) || defined(__alpha)
+#if defined(__arch64__) || defined(__alpha) || defined(__x86_64)
 /* 64 bit architectures */
 
 typedef signed char     SBYTE;      /* 1 byte, signed */
diff --recursive --unified libmikmod-3.1.11/include/mikmod_internals.h libmikmod-3.1.11-a/include/mikmod_internals.h
--- libmikmod-3.1.11/include/mikmod_internals.h	2004-01-21 12:43:53.000000000 -0500
+++ libmikmod-3.1.11-a/include/mikmod_internals.h	2004-06-04 08:24:26.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: mikmod_internals.h,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: mikmod_internals.h,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
 
   MikMod sound library internal definitions
 
diff --recursive --unified libmikmod-3.1.11/libmikmod.m4 libmikmod-3.1.11-a/libmikmod.m4
--- libmikmod-3.1.11/libmikmod.m4	2004-01-21 12:43:53.000000000 -0500
+++ libmikmod-3.1.11-a/libmikmod.m4	2004-06-04 08:24:26.000000000 -0400
@@ -8,7 +8,7 @@
 dnl Test for libmikmod, and define LIBMIKMOD_CFLAGS, LIBMIKMOD_LIBS and
 dnl LIBMIKMOD_LDADD
 dnl
-AC_DEFUN(AM_PATH_LIBMIKMOD,
+AC_DEFUN([AM_PATH_LIBMIKMOD],
 [dnl 
 dnl Get the cflags and libraries from the libmikmod-config script
 dnl
diff --recursive --unified libmikmod-3.1.11/loaders/load_669.c libmikmod-3.1.11-a/loaders/load_669.c
--- libmikmod-3.1.11/loaders/load_669.c	2004-01-21 12:43:53.000000000 -0500
+++ libmikmod-3.1.11-a/loaders/load_669.c	2004-06-04 08:24:27.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: load_669.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: load_669.c,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
 
   Composer 669 module loader
 
diff --recursive --unified libmikmod-3.1.11/loaders/load_amf.c libmikmod-3.1.11-a/loaders/load_amf.c
--- libmikmod-3.1.11/loaders/load_amf.c	2004-01-21 12:43:53.000000000 -0500
+++ libmikmod-3.1.11-a/loaders/load_amf.c	2004-06-04 08:24:27.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: load_amf.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: load_amf.c,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
 
   DMP Advanced Module Format loader
 
diff --recursive --unified libmikmod-3.1.11/loaders/load_dsm.c libmikmod-3.1.11-a/loaders/load_dsm.c
--- libmikmod-3.1.11/loaders/load_dsm.c	2004-01-21 12:43:53.000000000 -0500
+++ libmikmod-3.1.11-a/loaders/load_dsm.c	2004-06-04 08:24:27.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: load_dsm.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: load_dsm.c,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
 
   DSIK internal format (DSM) module loader
 
diff --recursive --unified libmikmod-3.1.11/loaders/load_far.c libmikmod-3.1.11-a/loaders/load_far.c
--- libmikmod-3.1.11/loaders/load_far.c	2004-01-21 12:43:53.000000000 -0500
+++ libmikmod-3.1.11-a/loaders/load_far.c	2004-06-04 08:24:27.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: load_far.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: load_far.c,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
 
   Farandole (FAR) module loader
 
diff --recursive --unified libmikmod-3.1.11/loaders/load_gdm.c libmikmod-3.1.11-a/loaders/load_gdm.c
--- libmikmod-3.1.11/loaders/load_gdm.c	2004-01-21 12:43:53.000000000 -0500
+++ libmikmod-3.1.11-a/loaders/load_gdm.c	2004-06-04 08:24:27.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: load_gdm.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: load_gdm.c,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
 
   General DigiMusic (GDM) module loader
 
diff --recursive --unified libmikmod-3.1.11/loaders/load_imf.c libmikmod-3.1.11-a/loaders/load_imf.c
--- libmikmod-3.1.11/loaders/load_imf.c	2004-01-21 12:43:53.000000000 -0500
+++ libmikmod-3.1.11-a/loaders/load_imf.c	2004-06-04 08:24:27.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: load_imf.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: load_imf.c,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
 
   Imago Orpheus (IMF) module loader
 
diff --recursive --unified libmikmod-3.1.11/loaders/load_it.c libmikmod-3.1.11-a/loaders/load_it.c
--- libmikmod-3.1.11/loaders/load_it.c	2004-01-21 12:43:52.000000000 -0500
+++ libmikmod-3.1.11-a/loaders/load_it.c	2004-06-04 08:24:27.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: load_it.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: load_it.c,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
 
   Impulse tracker (IT) module loader
 
diff --recursive --unified libmikmod-3.1.11/loaders/load_m15.c libmikmod-3.1.11-a/loaders/load_m15.c
--- libmikmod-3.1.11/loaders/load_m15.c	2004-01-21 12:43:53.000000000 -0500
+++ libmikmod-3.1.11-a/loaders/load_m15.c	2004-06-04 08:24:27.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: load_m15.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: load_m15.c,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
 
   15 instrument MOD loader
   Also supports Ultimate Sound Tracker (old M15 format)
diff --recursive --unified libmikmod-3.1.11/loaders/load_med.c libmikmod-3.1.11-a/loaders/load_med.c
--- libmikmod-3.1.11/loaders/load_med.c	2004-01-21 12:43:53.000000000 -0500
+++ libmikmod-3.1.11-a/loaders/load_med.c	2004-06-04 08:24:27.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: load_med.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: load_med.c,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
 
   Amiga MED module loader
 
diff --recursive --unified libmikmod-3.1.11/loaders/load_mod.c libmikmod-3.1.11-a/loaders/load_mod.c
--- libmikmod-3.1.11/loaders/load_mod.c	2004-01-21 12:43:53.000000000 -0500
+++ libmikmod-3.1.11-a/loaders/load_mod.c	2004-06-04 08:24:27.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: load_mod.c,v 1.2 2004/01/21 13:33:11 raph Exp $
+  $Id: load_mod.c,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
 
   Generic MOD loader (Protracker, StarTracker, FastTracker, etc)
 
diff --recursive --unified libmikmod-3.1.11/loaders/load_mtm.c libmikmod-3.1.11-a/loaders/load_mtm.c
--- libmikmod-3.1.11/loaders/load_mtm.c	2004-01-21 12:43:53.000000000 -0500
+++ libmikmod-3.1.11-a/loaders/load_mtm.c	2004-06-04 08:24:27.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: load_mtm.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: load_mtm.c,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
 
   MTM module loader
 
diff --recursive --unified libmikmod-3.1.11/loaders/load_okt.c libmikmod-3.1.11-a/loaders/load_okt.c
--- libmikmod-3.1.11/loaders/load_okt.c	2004-01-21 12:43:53.000000000 -0500
+++ libmikmod-3.1.11-a/loaders/load_okt.c	2004-06-04 08:24:27.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: load_okt.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: load_okt.c,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
 
   Oktalyzer (OKT) module loader
 
diff --recursive --unified libmikmod-3.1.11/loaders/load_s3m.c libmikmod-3.1.11-a/loaders/load_s3m.c
--- libmikmod-3.1.11/loaders/load_s3m.c	2004-01-21 12:43:53.000000000 -0500
+++ libmikmod-3.1.11-a/loaders/load_s3m.c	2004-06-04 08:24:27.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: load_s3m.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: load_s3m.c,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
 
   Screamtracker (S3M) module loader
 
diff --recursive --unified libmikmod-3.1.11/loaders/load_stm.c libmikmod-3.1.11-a/loaders/load_stm.c
--- libmikmod-3.1.11/loaders/load_stm.c	2004-01-21 12:43:53.000000000 -0500
+++ libmikmod-3.1.11-a/loaders/load_stm.c	2004-06-04 08:24:27.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: load_stm.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: load_stm.c,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
 
   Screamtracker 2 (STM) module loader
 
diff --recursive --unified libmikmod-3.1.11/loaders/load_stx.c libmikmod-3.1.11-a/loaders/load_stx.c
--- libmikmod-3.1.11/loaders/load_stx.c	2004-01-21 12:43:53.000000000 -0500
+++ libmikmod-3.1.11-a/loaders/load_stx.c	2004-06-04 08:24:27.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: load_stx.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: load_stx.c,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
 
   STMIK 0.2 (STX) module loader
 
diff --recursive --unified libmikmod-3.1.11/loaders/load_ult.c libmikmod-3.1.11-a/loaders/load_ult.c
--- libmikmod-3.1.11/loaders/load_ult.c	2004-01-21 12:43:53.000000000 -0500
+++ libmikmod-3.1.11-a/loaders/load_ult.c	2004-06-04 08:24:27.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: load_ult.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: load_ult.c,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
 
   Ultratracker (ULT) module loader
 
diff --recursive --unified libmikmod-3.1.11/loaders/load_uni.c libmikmod-3.1.11-a/loaders/load_uni.c
--- libmikmod-3.1.11/loaders/load_uni.c	2004-01-21 12:43:53.000000000 -0500
+++ libmikmod-3.1.11-a/loaders/load_uni.c	2004-06-04 08:24:27.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: load_uni.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: load_uni.c,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
 
   UNIMOD (libmikmod's and APlayer's internal module format) loader
 
diff --recursive --unified libmikmod-3.1.11/loaders/load_xm.c libmikmod-3.1.11-a/loaders/load_xm.c
--- libmikmod-3.1.11/loaders/load_xm.c	2004-01-21 12:43:53.000000000 -0500
+++ libmikmod-3.1.11-a/loaders/load_xm.c	2004-06-04 08:24:27.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: load_xm.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: load_xm.c,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
 
   Fasttracker (XM) module loader
 
diff --recursive --unified libmikmod-3.1.11/macintosh/mikmod.h libmikmod-3.1.11-a/macintosh/mikmod.h
--- libmikmod-3.1.11/macintosh/mikmod.h	2004-01-21 12:43:53.000000000 -0500
+++ libmikmod-3.1.11-a/macintosh/mikmod.h	2004-06-04 08:24:27.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: mikmod.h,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: mikmod.h,v 1.1.1.1 2004/06/01 12:16:18 raph Exp $
 
   MikMod sound library include file
 
diff --recursive --unified libmikmod-3.1.11/mkinstalldirs libmikmod-3.1.11-a/mkinstalldirs
--- libmikmod-3.1.11/mkinstalldirs	2004-01-21 12:43:53.000000000 -0500
+++ libmikmod-3.1.11-a/mkinstalldirs	2004-06-04 08:24:26.000000000 -0400
@@ -4,7 +4,7 @@
 # Created: 1993-05-16
 # Public domain
 
-# $Id: mkinstalldirs,v 1.1.1.1 2004/01/21 01:36:34 raph Exp $
+# $Id: mkinstalldirs,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
 
 errstatus=0
 
diff --recursive --unified libmikmod-3.1.11/mmio/mmalloc.c libmikmod-3.1.11-a/mmio/mmalloc.c
--- libmikmod-3.1.11/mmio/mmalloc.c	2004-01-21 12:43:52.000000000 -0500
+++ libmikmod-3.1.11-a/mmio/mmalloc.c	2004-06-04 08:24:27.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: mmalloc.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: mmalloc.c,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
 
   Dynamic memory routines
 
diff --recursive --unified libmikmod-3.1.11/mmio/mmerror.c libmikmod-3.1.11-a/mmio/mmerror.c
--- libmikmod-3.1.11/mmio/mmerror.c	2004-01-21 12:43:52.000000000 -0500
+++ libmikmod-3.1.11-a/mmio/mmerror.c	2004-06-04 08:24:27.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: mmerror.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: mmerror.c,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
 
   Error handling functions.
   Register an error handler with _mm_RegisterErrorHandler() and you're all set.
diff --recursive --unified libmikmod-3.1.11/mmio/mmio.c libmikmod-3.1.11-a/mmio/mmio.c
--- libmikmod-3.1.11/mmio/mmio.c	2004-01-21 12:43:52.000000000 -0500
+++ libmikmod-3.1.11-a/mmio/mmio.c	2004-06-04 08:24:27.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: mmio.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: mmio.c,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
 
   Portable file I/O routines
 
diff --recursive --unified libmikmod-3.1.11/os2/configure.cmd libmikmod-3.1.11-a/os2/configure.cmd
--- libmikmod-3.1.11/os2/configure.cmd	2004-01-21 12:43:52.000000000 -0500
+++ libmikmod-3.1.11-a/os2/configure.cmd	2004-06-04 08:24:27.000000000 -0400
@@ -22,7 +22,7 @@
 
 /*==============================================================================
 
-  $Id: configure.cmd,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: configure.cmd,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
 
   Configuration script for libmikmod under OS/2
 
diff --recursive --unified libmikmod-3.1.11/playercode/mdreg.c libmikmod-3.1.11-a/playercode/mdreg.c
--- libmikmod-3.1.11/playercode/mdreg.c	2004-01-21 12:43:53.000000000 -0500
+++ libmikmod-3.1.11-a/playercode/mdreg.c	2004-06-04 08:24:27.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: mdreg.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: mdreg.c,v 1.1.1.1 2004/06/01 12:16:18 raph Exp $
 
   Routine for registering all drivers in libmikmod for the current platform.
 
diff --recursive --unified libmikmod-3.1.11/playercode/mdriver.c libmikmod-3.1.11-a/playercode/mdriver.c
--- libmikmod-3.1.11/playercode/mdriver.c	2004-01-21 12:43:53.000000000 -0500
+++ libmikmod-3.1.11-a/playercode/mdriver.c	2004-06-04 08:24:27.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: mdriver.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: mdriver.c,v 1.1.1.1 2004/06/01 12:16:18 raph Exp $
 
   These routines are used to access the available soundcard drivers.
 
diff --recursive --unified libmikmod-3.1.11/playercode/mdulaw.c libmikmod-3.1.11-a/playercode/mdulaw.c
--- libmikmod-3.1.11/playercode/mdulaw.c	2004-01-21 12:43:53.000000000 -0500
+++ libmikmod-3.1.11-a/playercode/mdulaw.c	2004-06-04 08:24:27.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: mdulaw.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: mdulaw.c,v 1.1.1.1 2004/06/01 12:16:18 raph Exp $
 
   Mu-law routines
 
diff --recursive --unified libmikmod-3.1.11/playercode/mloader.c libmikmod-3.1.11-a/playercode/mloader.c
--- libmikmod-3.1.11/playercode/mloader.c	2004-01-21 12:43:53.000000000 -0500
+++ libmikmod-3.1.11-a/playercode/mloader.c	2004-06-04 08:24:27.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: mloader.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: mloader.c,v 1.1.1.1 2004/06/01 12:16:18 raph Exp $
 
   These routines are used to access the available module loaders
 
diff --recursive --unified libmikmod-3.1.11/playercode/mlreg.c libmikmod-3.1.11-a/playercode/mlreg.c
--- libmikmod-3.1.11/playercode/mlreg.c	2004-01-21 12:43:53.000000000 -0500
+++ libmikmod-3.1.11-a/playercode/mlreg.c	2004-06-04 08:24:27.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: mlreg.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: mlreg.c,v 1.1.1.1 2004/06/01 12:16:18 raph Exp $
 
   Routine for registering all loaders in libmikmod for the current platform.
 
diff --recursive --unified libmikmod-3.1.11/playercode/mlutil.c libmikmod-3.1.11-a/playercode/mlutil.c
--- libmikmod-3.1.11/playercode/mlutil.c	2004-01-21 12:43:53.000000000 -0500
+++ libmikmod-3.1.11-a/playercode/mlutil.c	2004-06-04 08:24:27.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: mlutil.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: mlutil.c,v 1.1.1.1 2004/06/01 12:16:18 raph Exp $
 
   Utility functions for the module loader
 
diff --recursive --unified libmikmod-3.1.11/playercode/mplayer.c libmikmod-3.1.11-a/playercode/mplayer.c
--- libmikmod-3.1.11/playercode/mplayer.c	2004-01-21 12:43:53.000000000 -0500
+++ libmikmod-3.1.11-a/playercode/mplayer.c	2004-06-04 08:24:27.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: mplayer.c,v 1.2 2004/01/21 01:41:55 raph Exp $
+  $Id: mplayer.c,v 1.1.1.1 2004/06/01 12:16:18 raph Exp $
 
   The Protracker Player Driver
 
diff --recursive --unified libmikmod-3.1.11/playercode/munitrk.c libmikmod-3.1.11-a/playercode/munitrk.c
--- libmikmod-3.1.11/playercode/munitrk.c	2004-01-21 12:43:53.000000000 -0500
+++ libmikmod-3.1.11-a/playercode/munitrk.c	2004-06-04 08:24:27.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: munitrk.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: munitrk.c,v 1.1.1.1 2004/06/01 12:16:18 raph Exp $
 
   All routines dealing with the manipulation of UNITRK streams
 
diff --recursive --unified libmikmod-3.1.11/playercode/mwav.c libmikmod-3.1.11-a/playercode/mwav.c
--- libmikmod-3.1.11/playercode/mwav.c	2004-01-21 12:43:53.000000000 -0500
+++ libmikmod-3.1.11-a/playercode/mwav.c	2004-06-04 08:24:27.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: mwav.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: mwav.c,v 1.1.1.1 2004/06/01 12:16:18 raph Exp $
 
   WAV sample loader
 
diff --recursive --unified libmikmod-3.1.11/playercode/npertab.c libmikmod-3.1.11-a/playercode/npertab.c
--- libmikmod-3.1.11/playercode/npertab.c	2004-01-21 12:43:53.000000000 -0500
+++ libmikmod-3.1.11-a/playercode/npertab.c	2004-06-04 08:24:27.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: npertab.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: npertab.c,v 1.1.1.1 2004/06/01 12:16:18 raph Exp $
 
   MOD format period table.  Used by both the MOD and M15 (15-inst mod) Loaders.
 
diff --recursive --unified libmikmod-3.1.11/playercode/sloader.c libmikmod-3.1.11-a/playercode/sloader.c
--- libmikmod-3.1.11/playercode/sloader.c	2004-01-21 12:43:53.000000000 -0500
+++ libmikmod-3.1.11-a/playercode/sloader.c	2004-06-04 08:24:27.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: sloader.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: sloader.c,v 1.1.1.1 2004/06/01 12:16:18 raph Exp $
 
   Routines for loading samples. The sample loader utilizes the routines
   provided by the "registered" sample loader.
diff --recursive --unified libmikmod-3.1.11/playercode/virtch.c libmikmod-3.1.11-a/playercode/virtch.c
--- libmikmod-3.1.11/playercode/virtch.c	2004-01-21 12:43:53.000000000 -0500
+++ libmikmod-3.1.11-a/playercode/virtch.c	2004-06-04 08:24:27.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: virtch.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: virtch.c,v 1.1.1.1 2004/06/01 12:16:18 raph Exp $
 
   Sample mixing routines, using a 32 bits mixing buffer.
 
diff --recursive --unified libmikmod-3.1.11/playercode/virtch2.c libmikmod-3.1.11-a/playercode/virtch2.c
--- libmikmod-3.1.11/playercode/virtch2.c	2004-01-21 12:43:53.000000000 -0500
+++ libmikmod-3.1.11-a/playercode/virtch2.c	2004-06-04 08:24:27.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: virtch2.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: virtch2.c,v 1.1.1.1 2004/06/01 12:16:18 raph Exp $
 
   High-quality sample mixing routines, using a 32 bits mixing buffer,
   interpolation, and sample smoothing to improve sound quality and remove
diff --recursive --unified libmikmod-3.1.11/playercode/virtch_common.c libmikmod-3.1.11-a/playercode/virtch_common.c
--- libmikmod-3.1.11/playercode/virtch_common.c	2004-01-21 12:43:53.000000000 -0500
+++ libmikmod-3.1.11-a/playercode/virtch_common.c	2004-06-04 08:24:27.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: virtch_common.c,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: virtch_common.c,v 1.1.1.1 2004/06/01 12:16:18 raph Exp $
 
   Common source parts between the two software mixers.
   This file is probably the ugliest part of libmikmod...
diff --recursive --unified libmikmod-3.1.11/win32/mikmod_build.h libmikmod-3.1.11-a/win32/mikmod_build.h
--- libmikmod-3.1.11/win32/mikmod_build.h	2004-01-21 12:43:52.000000000 -0500
+++ libmikmod-3.1.11-a/win32/mikmod_build.h	2004-06-04 08:24:27.000000000 -0400
@@ -20,7 +20,7 @@
 
 /*==============================================================================
 
-  $Id: mikmod_build.h,v 1.1.1.1 2004/01/21 01:36:35 raph Exp $
+  $Id: mikmod_build.h,v 1.1.1.1 2004/06/01 12:16:17 raph Exp $
 
   MikMod sound library include file
 
